packages/components/src/components/bal-list/bal-list-item-accordion-body/bal-list-item-accordion-body.tsx

```diff
@@ -20,6 +20,11 @@ export class ListItemAccordionBody {
     this.calcContentHeight()
   }
 
+  @Listen('balAccordionChange', { target: 'window' })
```

Comment:
do we also need to this.calcContentHeight on a window resize?

Reply:
Yes we do need it on resize listener. But I removed the here-highlighted change listener